chimney/.jvm/src/test/scala/io/scalaland/chimney/WixSpec.scala

```diff
@@ -4,8 +4,14 @@ import io.scalaland.chimney.dsl._
 import io.scalaland.chimney.examples._
 import io.scalaland.chimney.examples.wix._
 import JavaColors.{Colors, ColorsUpperCase}
-import io.scalaland.chimney.internal.{CoproductInstanceNotFoundException, TransformerCfg, TransformerFlags}
+import io.scalaland.chimney.internal.{
```

Comment:
nit: organize imports in idea and then do scalafmt. scalafmt doesn't reorganize imports